salt/defaults.yaml

```
@@ -30,11 +30,16 @@ salt:
   salt_api: salt-api
   salt_ssh: salt-ssh
 
+  pkgrepo: ''          # see osfamilymap
+  ssh_roster: ''       # see pillar data
```

To match the actual definition type:

```suggestion
  ssh_roster: {}       # see pillar data
```

---

okay, thanks.